packages/core/src/components/Onboarding/index.tsx

```diff
@@ -36,6 +37,12 @@ export function Onboarding({
 
   const [page, setPage] = useState<number>(platform === 'web' ? 1 : 0)
 
+  const updatePrimaryColor = (newColor: string) => {
+    setPrimaryColor(newColor)
+    const names = namer(newColor)
+    setPrimaryName(names?.html[0]?.name)
```

Comment:
Haven't tested it out, but will my previous name be overwritten by a default one if I change a color?  And if it does, is that expected?

Reply:
Yes the user can change the default color suggested if they want to. They should be able to choose their preferred color name.